themes/HeroesLounge-Theme/partials/user/standings.htm

```diff
@@ -3,18 +3,37 @@
 
 [viewBag]
 ==
-{% if user.sloth.team  and user.sloth.team.active_divisions | length > 0 %}
+{% if user.sloth.teams|length > 0 %}
 <div class="col-md-6 col-12">
-    <h3>Standings</h3>
-    {% for div in user.sloth.team.active_divisions %}
-        {% if div.season %}
-        <h4><a href="{{ 'season/division'|page({slug: div.season.slug, divslug: div.slug})}}"> {{div.title}}</a></h4>
-        {% elseif div.playoff %}
-        <h4><a href="{{ 'playoff/view'|page({'playoff-title': div.playoff.title})}}"> {{div.title}}</a></h4>
-        {% else %}
-        <h4> {{div.title}} </h4>
+    <ul class="nav nav-tabs">
+        {% for team in user.sloth.teams %}
+        {% if team.active_divisions | length > 0 %}
```

Comment:
ditto

Reply:
Actually, this can just be deleted since its followed by a for directly afterwards